IdentityCore/src/cache/accessor/MSIDAccountCredentialCache.m

```diff
@@ -165,9 +166,10 @@ - (nullable MSIDCredentialCacheItem *)getCredential:(nonnull MSIDDefaultCredenti
         for (MSIDAccountCacheItem *cacheItem in cacheItems)
         {
             if (shouldMatchAccount
-                && ![cacheItem matchesWithHomeAccountId:cacheQuery.homeAccountId
-                                           environment:cacheQuery.environment
-                                    environmentAliases:cacheQuery.environmentAliases])
+                && ![MSIDCacheItemUtils accountMatchesWithAccount:cacheItem
```

Comment:
Didn't we have a category for this matching introduced by Dean?

Reply:
We ended up not committing this since we found a way to avoid needing the change at the time.